From: Cole Robinson <crobinso@redhat.com>
Date: Wed, 2 Nov 2016 10:27:14 -0400
Subject: [PATCH virt-manager] virt-install: Fix error checking extra_args

Later bits in the code that want to warn based on extra_args content
don't handle the None case. Be consistent and convert it to a list
everywhere.

Mentioned at https://bugzilla.redhat.com/show_bug.cgi?id=1376547#c9

(cherry picked from commit 7962672c713cf6d35e770f0d00068dee707b6ec9)
---
 virt-install | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/virt-install b/virt-install
index 817f4b3..5a4080f 100755
--- a/virt-install
+++ b/virt-install
@@ -595,7 +595,8 @@ def build_guest_instance(conn, options):
     convert_old_os_options(options)
 
     # non-xml install options
-    guest.installer.extraargs = options.extra_args or []
+    options.extra_args = options.extra_args or []
+    guest.installer.extraargs = options.extra_args
     guest.installer.initrd_injections = options.initrd_inject
     guest.autostart = options.autostart
 
